### PR TITLE
Document Apache 2.4+

### LINKED ISSUE #5790: [BUG] Freshrss 1.22.1 , 500 internal server errror while starting installation on Apache/2.2.34
**Describe the bug**

Hello,

I was in the process of manual installation of Freshrss 1.22.1 on Centos 7 with  PHP 7.4, Apache 2.2.34, MYSQL 5.5. 

However I see a blank page with 500 internal server error for URL http://example.com/freshrss1221/p/i/ under browser's network console .

So to debug further I checked with Apache logs and it shows : 

`/{{PATH}}/{{TO}}/{{FRESHRSS}}/p/.htaccess: Invalid command 'SetEnvIfExpr', perhaps misspelled or defined by a module not included in the server configuration.`

What could be the issue ?. 

Could you please replicate this on your server with Apache 2.2 and fix this? . 

I checked the requirements here in the following URL'S and it's mentioned that script  is compatible with Apache 2  : 

https://freshrss.github.io/FreshRSS/en/admins/02_Prerequisites.html
https://github.com/FreshRSS/FreshRSS#requirements

**To Reproduce**
1. Go to 'http://example.com/freshrss1221/p/i/'
2. See error in browser console

**Expected behavior**
Freshrss installation should have been started.

**Environment information (please complete the following information):**
- OS: [e.g. Ubuntu 20.04] - CentOS Linux release 7.6.1810 (Core)
- Browser: [e.g. Firefox 86] - Firefox 115.4.0esr (64-bit)
- FreshRSS version: [e.g. 1.17.1] - 1.22.1
- Database version: [e.g. Mysql 5.7] - MYSQL 5.5
- PHP version: [e.g. PHP 7.4] - PHP 7.4.25
- Installation type: Shared hosting

### PR DESCRIPTION
**Pull request by Alkarex**
And avoid crash in Apache 2.2 anyway
fix https://github.com/FreshRSS/FreshRSS/issues/5790

### CODE REVIEW

**Review by math-GH — APPROVED** [review]

File: p/.htaccess [hunk 1]
```diff
@@ -38,7 +38,10 @@ AddDefaultCharset	UTF-8
 
 # Provide the true IP address of the connection (e.g. last proxy), even when using mod_remoteip
 <IfModule mod_setenvif.c>
-	SetEnvIfExpr "%{CONN_REMOTE_ADDR} =~ /(.*)/" CONN_REMOTE_ADDR=$1
+	<IfVersion >= 2.4>
+		# If you run an old Apache 2.2-, disable mod_setenvif and enable mod_rewrite, or edit manually
+		SetEnvIfExpr "%{CONN_REMOTE_ADDR} =~ /(.*)/" CONN_REMOTE_ADDR=$1
+	</IfVersion>
```
Comment (math-GH) [inline p/.htaccess:44]: I updated from 1.22.1-dev to 1.22.2-dev and got this error that leads to a HTTP500 Internal Server Error: 
`/freshRSS/p/.htaccess: Invalid command '<IfVersion', perhaps misspelled or defined by a module not included in the server configuration`

Comment (Alkarex) [reply p/.htaccess:44]: Damn, `IfVersion` was supposed to be available by default in most systems. Will change

Comment (Alkarex) [reply p/.htaccess:44]: https://github.com/FreshRSS/FreshRSS/issues/5803

Comment (Alkarex) [reply p/.htaccess:44]: Please try https://github.com/FreshRSS/FreshRSS/pull/5804

**Pull request merged**